MachOKit/Core/MKNode.m

```diff
@@ -197,13 +197,6 @@ + (NSSet*)subclasses
         
         if (cache)
             objc_storeWeak(cache, retValue);
-        
-        // We autorelease the returned set in order to force our cached
-        // subclasses to be flushed when the autorelease pool next drains.
-        // We can't cache the subclasses for too long because our client may
-        // load additional bundles or frameworks at runtime which may augment
-        // the available subclasses.
-        [retValue autorelease];
```

Comment:
I'm fine with this, and I trust your performance numbers.  I do want a mechanism for flushing the cache(s) when a new framework/bundle is loaded.  I like your suggestion for a `+[MKNode flushSubclassCache]` method.  The caching storage will need to be re-worked: get rid of `-[MKNode _subclassesCache]` with `__weak` storage and have `MKNode` instead keep a dictionary of subclass => cache that can quickly be cleared on cache flush.

I can handle that change after this PR is merged.